components/signUp/signUp.css

```diff
@@ -0,0 +1,100 @@
+@import url('../../style.css');
+
+.login-container{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
+    margin-top: 1.5rem;
+    background: var(--gray-bg-color);
+}
+
+.login-container h3{
+    padding-top: 1rem;
+    font-size: 1.8rem;
+}
+
+.login-card{
+    display: flex;
+    flex-direction: column;
+    justify-content:center;
+    align-items: center;
```

Comment:
You can plan to use DRY principle. Here in login-container and login-card classes the flex properties are repeating. So you can make a separate class for that altogether.